akka-docs/src/main/paradox/project/migration-guide-2.5.x-2.6.x.md

```diff
@@ -195,6 +195,11 @@ Classic remoting is deprecated but can be used in `2.6.` Explicitly disable Arte
 specific to classic remoting needs to be moved to `akka.remote.classic`. To see which configuration options
 are specific to classic search for them in: [`akka-remote/reference.conf`](/akka-remote/src/main/resources/reference.conf)
 
+### Cluster client
+
+Cluster client has been deprecated as of 2.6 in favor of [Akka gRPC](https://doc.akka.io/docs/akka-grpc/current/index.html).
+It is not advised to build new applications with Cluster client, and existing users should migrate (FIXME https://github.com/akka/akka/issues/26175) to Akka gRPC.
+
```

Comment:
The FIXME is the actual migration example and recommendation @patriknw started in https://github.com/patriknw/akka-cluster-client-grpc/pull/2 needing reviews. This is just the entry of the change. When the samples are merged it should be added as links here.